ja/feature-nonhttp.md

```diff
@@ -1,3 +1,3 @@
 ## HTTP 以外のプロトコルを QUIC 上で動かす
 
-HTTP 以外のプロトコルを QUIC を使って通信するための対応は、QUIC のバージョン1が実際にリリースされた後に行われるよう延期されました。
\ No newline at end of file
+HTTP 以外のプロトコルを QUIC を使って通信するための対応は、QUIC のバージョン1が実際にリリースされた後に行われるよう延期されました。
```

Comment:
Why changed?

Reply:
There is no `newline` at the end of file in this file.
(All other `ja/*.md` have `newline` at the end of file.)